### PR TITLE
ISPN-12433 Fix delete cache latest page

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12433

### CODE REVIEW

**Comment by oraNod** [comment]
@karesti it looks like there are conflicts. can you fix?

**Comment by karesti** [comment]
@oraNod rebased

File: src/services/dataContainerService.ts [hunk 1]
```diff
@@ -24,32 +24,14 @@ class ContainerService {
...
-        return left(<ActionResponse>{
-          message: errorMessage,
-          success: false,
-        });
-      });
+      .catch((err) =>
+        left(
+          utils.mapError(
+            err,
+            'Failed to connect, check the navigator logs please.'
```
Comment (oraNod) [inline src/services/dataContainerService.ts:31]:
```suggestion
            'Cannot connect. Check the navigator logs for errors.'
```

File: src/services/dataContainerService.ts [hunk 2]
```diff
@@ -104,50 +86,49 @@ class ContainerService {
...
-        arr.map(
-          (config) =>
-            <CacheConfig>{
-              name: config.name,
-              config: JSON.stringify(config.configuration, undefined, 2),
-            }
-        )
+      .then((data) => right(<CacheManagerStats>data))
+      .catch((err) =>
+        left(utils.mapError(err, 'Error retrieving cache manager stats'))
```
Comment (oraNod) [inline src/services/dataContainerService.ts:102]:
```suggestion
        left(utils.mapError(err, 'Error retrieving cache manager statistics.'))
```

File: src/services/dataContainerService.ts [hunk 3]
```diff
@@ -104,50 +86,49 @@ class ContainerService {
...
             <CacheConfig>{
               name: config.name,
               config: JSON.stringify(config.configuration, undefined, 2),
             }
-        )
+        );
+        return right(configs);
+      })
+      .catch((err) =>
+        left(utils.mapError(err, 'Error retrieving configuration templates'))
```
Comment (oraNod) [inline src/services/dataContainerService.ts:131]:
```suggestion
        left(utils.mapError(err, 'Error retrieving configuration templates.'))
```

File: src/services/dataContainerService.ts [hunk 4]
```diff
@@ -186,13 +167,7 @@ class ContainerService {
...
         )
       )
-      .catch((err) =>
-        err
-          .text()
-          .then((errorMessage) =>
-            left(<ActionResponse>{ message: errorMessage, success: false })
-          )
-      );
+      .catch((err) => left(utils.mapError(err, 'Error retrieving caches')));
```
Comment (oraNod) [inline src/services/dataContainerService.ts:170]:
```suggestion
      .catch((err) => left(utils.mapError(err, 'Error retrieving caches.')));
```

File: src/services/utils.ts [hunk 5]
```diff
@@ -217,6 +218,15 @@ class Utils {
       };
     }
 
+    if (err instanceof Response) {
+      if (err.status == 401) {
+        return <ActionResponse>{
+          message: 'Unauthorized. You need to provide the correct credentials.',
```
Comment (oraNod) [inline src/services/utils.ts:224]:
```suggestion
          message: 'Login failed. Check your credentials and try again.',
```

**Comment by karesti** [comment]
squased @oraNod thanks!!!

**Comment by oraNod** [comment]
thanks @karesti

**Pull request merged**